src/styles.scss

```diff
@@ -334,4 +334,8 @@ body {
   mat-dialog-actions {
     justify-content: flex-end;
   }
+
+  .mat-raised-button {

```

Comment:
This is overriding margins added via other classes to some buttons.  Let's just have this as part of the component style.

Reply:
What do you mean by that @paulbert? 
Only adding it to the users component?